packages/messaging/utils/messenger.ts

```diff
@@ -67,11 +67,11 @@ export const createMessenger = (
 	) => {
 		const revealed = await smartReveal(data, channel.name);
 		const requestId = revealed?.requestId;
-		const associatedRequest = requestHashmap[requestId];
+		const associatedRequest = requestHashmap[requestId as string];
```

Comment:
Cast `as string` once at `const requestId = revealed?.requestId as string;` instead of multiple casts.